test/influx-integration.spec.js

```diff
@@ -435,7 +435,7 @@ describe('@integration influxdb tests', () => {
 
             it('successfully retrieves live points', () => {
                 return check_juttle({
-                    program: 'read influx -db "test" -from :now: -to :end: -every :0.1s: name = "live" | view logger',
+                    program: 'read influx -db "test" -to :end: -every :0.1s: name = "live" | view logger',
```

Comment:
i don't follow, you removed the explicit `-from :now:` but that's the default, it should be equivalent?